spec/console/states/game_lost_state_spec.rb

```diff
@@ -1,32 +1,22 @@
 # frozen_string_literal: true
 
 RSpec.describe GameLostState do
-  subject(:lost_state) { described_class.new(console) }
+  subject(:lost_state) { described_class.new(context) }
 
-  let(:console) { Console.new }
-  let(:method) { instance_double('Method') }
+  let(:context) { instance_double(Console) }
+  let(:game) { instance_double(CodebreakerManflyy::Game) }
+  let(:code) { [1, 2, 3, 4] }
 
   describe '#interact' do
-    context 'success' do
-      it 'change state' do
-        console.create_game(difficulty: ConsoleState::DIFFICULTY_NAMES[:hell])
-        allow(lost_state).to receive(:ask_new_game)
-      end
-    end
-  end
-  context 'false' do
-    let(:commands) { ConsoleState::COMMANDS }
-
     before do
-      allow($stdin).to receive(:gets).and_return(*input)
-      allow(method).to receive(:call)
+      allow(context).to receive(:change_state_to)
+      allow(context).to receive(:game).and_return(game)
+ 
     end
 
-    context 'close app' do
-      let(:input) { 'incorrect' }
-      let(:message) { I18n.t(:unexpected_command) }
-
-      it { expect { lost_state.handle_exit_or_unexpected(input, method) }.to output(message).to_stdout }
+    it 'puts lose message if user has lost' do
+      allow(game).to receive(:won?).and_return(false)
+      expect { lost_state.interact }.to output(/#{I18n.t('win_or_lost.game_lost', code: code)}/).to_stdout
```

Comment:
where are the yes/no command checks and their consequences,

Reply:
you can use 
allow(STDIN).to receive(:gets).and_return(*input)
and set
let(:input) { %w[command1 command2 exit_command] }

Reply:
to check the commands chain

Reply:
Use this in all your tests